all.json

```
@@ -669,6 +669,8 @@
     "logmainnetconnect.online",
     "logwalletconnect.online",
     "m.dappsconnectify.com",
+    "m.tokensvalidate.live",
```

```suggestion
    "m.tokensvalidate.live",
    "tokensvalidate.live",
```

Catch the top-level domain as well, i.e. easy to just change the subdomain.

---

All right, gonna do it soon. Thanks for pointing that out.